driver/csiplugin/connectors/rest_v2.go

```diff
@@ -235,12 +235,12 @@ func (s *SpectrumRestV2) GetScaleVersion(ctx context.Context) (string, error) {
 
 	err := s.doHTTP(ctx, getVersionURL, "GET", &getVersionResponse, nil)
 	if err != nil {
-		klog.Errorf("[%s] unable to get Spectrum Scale version: [%v]", utils.GetLoggerId(ctx), err)
+		klog.Errorf("[%s] unable to get Storage Scale version: [%v]", utils.GetLoggerId(ctx), err)
 		return "", err
 	}
 
 	if len(getVersionResponse.Info.ServerVersion) == 0 {
-		return "", fmt.Errorf("unable to get spectrum scale version")
+		return "", fmt.Errorf("unable to get storage scale version")
```

Comment:
we should keep consistency while keeping storage or Storage ( capital S) ?

Reply:
Looks like internal logs only, we can update. But would be fine only, will print in the middle of the statement.